docs/CONTRIBUTING.md

```diff
@@ -28,6 +28,10 @@ As of March 21, help is especially wanted from contributors with experience in K
 - Base your work on the `develop` branch.
 - Take a few minutes to review this [resource](https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Reviewer_Checklist) for contributors and reviewers, to accelerate the adoption of your contribution (thanks to the good folks at Mozilla for this).
 - Submit pull requests from your fork, also against the `develop` branch of the `CodeforPhilly/chime` repo.
+- As part of your pull request, please ensure you provide the following information
+  - Traceability of a change to a described issue (ex: Fixes issue # 100)
```

Comment:
Traceability is a rather fancy word, can we use something more straightforward? Eg "Link to any relevant issue(s)"